static/app/components/events/viewHierarchy/index.tsx

```diff
@@ -112,6 +115,11 @@ function ViewHierarchy({viewHierarchy, project}: ViewHierarchyProps) {
         onKeyDown={handleRowKeyDown}
         onClick={e => {
           handleRowClick(e);
+          trackAdvancedAnalyticsEvent('issue_details.view_hierarchy.select_from_tree', {
+            organization: project.organization,
```

Comment:
im sure this isn't an issue, but is there a reason why this analytic uses `project.organization` while the one below uses `organization` from the `useOrganization` hook?

Reply:
Ah, good catch. I thought I changed this, but I prefer to use the `organization` from `useOrganization`. Thanks!